data/wiki/hello-world.md

```diff
@@ -723,25 +723,20 @@ passed at compile time and runtime.
 
 ### Step 4: Networking
 
-There are several ways that we might want to configure our network for a Mirage
-application:
-
-* On Unix, it's convenient to use the standard kernel socket API for developing
-  higher level protocols (such
-  as [HTTP](http://github.com/mirage/ocaml-cohttp)). These run over TCP or UDP
-  and so sockets work just fine.
-* When we want finer control over the network stack, or simply to test the fully-OCaml
-  network implementation , we can use a userspace device facility such as the
-  common Unix [tuntap](http://en.wikipedia.org/wiki/TUN/TAP) to parse Ethernet
-  frames from userspace. This requires additional configuration to assign IP
-  addresses, and possibly configure a network bridge to let the unikernel talk
-  to the outside world.
-* Once the unikernel works under Unix with the
-  direct [OCaml TCP/IP stack](https://github.com/mirage/mirage-tcpip),
-  recompiling it for a unikernel target like `xen`, `hvt`, or `virtio` shouldn't
-  result in a change in behavior.
-
-All of this can be manipulated via command-line arguments or environment variables,
+You built multiple MirageOS unikernels, and at some point you may want them to
+communicate with the external world. We use network for that.
+
+Please note this document only explains how to communicate from your host system
+to your unikernel. For having your unikernel being able to communicate with the
+entire Internet, you will need to setup firewalling
+([NAT](https://en.wikipedia.org/wiki/Network_address_translation)).
+
+For MirageOS unikernels running as Unix application, we have the option to use
+the host system stack (Unix sockets API). For any other target, we must
+use the OCaml network stack. On Unix, we can also use the OCaml network stack
+by utilizing a [tuntap](http://en.wikipedia.org/wiki/TUN/TAP) interface.
+
+All of this can be manipulated via command-line arguments,
```

Comment:
This sounds like also the tap interface could be manipulated via command-line arguments to the unikernel. I assume that's not what you mean? What about the following?
```suggestion
Which of the two network stacks to use can be specified via command-line arguments,
```

Or do you mean that you can tell the unikernel the IP of the tap device? If so, what about:
```suggestion
Which of the two network stacks to use can be specified via command-line arguments. Also the IP of your tuntap interface can be passed in as a command-line argument.
```